docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -217,7 +217,7 @@ public class FruitResource {
 The implementation is pretty straightforward, and you just need to define your endpoints using the Jakarta REST annotations and use the `FruitService` to list/add new fruits.
 
 == Configuring the MongoDB database
-The main property to configure is the URL to access to MongoDB. Almost all configuration can be included in the connection URI, so we advise you to do so. You can find more information in the MongoDB documentation: https://docs.mongodb.com/manual/reference/connection-string/
+The main property to configure is the URL to access the MongoDB. Almost all configuration can be included in the connection URI, so we advise you to do so. You can find more information in the MongoDB documentation: https://docs.mongodb.com/manual/reference/connection-string/
```

Comment:
Thanks! I think it would be better with:

```suggestion
The main property to configure is the URL to access MongoDB. Almost all configuration can be included in the connection URI, so we advise you to do so. You can find more information in the MongoDB documentation: https://docs.mongodb.com/manual/reference/connection-string/
```

(or `the MongoDB server` or `the MongoDB instance`)

Could you adjust and squash?

Thanks!